### DISCUSSION TITLE
lcci/1.2/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#2304] [Announcements]
# lcci/1.2/

多种编程语言实现 LeetCode、《剑指 Offer（第 2 版）》、《程序员面试金典（第 6 版）》题解

https://leetcode.doocs.org/lcci/1.2/

**Comment by dayAndnight2018** [comment]
此处应该无法覆盖全部case，字符重拍中间部分的字母顺序是保持一致的，而不是乱序

**Reply by yanglbme** [reply]
哪个地方有问题，能否提供一下测试用例？
